packages/components/src/badge/stories/index.story.tsx

```diff
@@ -13,6 +13,22 @@ const meta = {
 	title: 'Components/Containers/Badge',
 	id: 'components-badge',
 	tags: [ 'status-private' ],
+	args: {
+		children: 'Code is Poetry',
+		intent: 'default',
+		// @ts-ignore
+		'...props': null,
+	},
+	argTypes: {
+		// @ts-ignore
+		'...props': {
```

Comment:
Maybe no string here is less confusing?

```suggestion
		'': {
```

I was also thinking of ways to force the row to be appended at the very end. Maybe something like this?

```suggestion
		'': {
					table: { category: 'Other' },
```

<img width="798" alt="Appended prop row" src="https://github.com/user-attachments/assets/c74e4e35-ffc7-471f-8853-10efa4294e3e" />